src/hardware/input/mouse_interfaces.h

```diff
@@ -40,9 +40,8 @@ bool MOUSE_IsProbeForMappingAllowed();
 // ***************************************************************************
 
 void MOUSEDOS_Init();
-void MOUSEDOS_SetDelay(const uint8_t new_delay_ms);
-void MOUSEDOS_NotifyInputType(const bool new_use_relative,
-                              const bool new_is_input_raw);
+void MOUSEDOS_SetDelay(const uint8_t in_delay_ms);
+void MOUSEDOS_NotifyInputType(const bool in_use_relative, const bool in_is_input_raw);
```

Comment:
`use_relative` and `is_input_raw` to remove ambiguity.

I read the first one as "in use relative". So I was thinking, if true this signals this is "in use", but then what's relative? `is_...` also is a bit clunky. Just drop these `in_` prefixes, they serve no purpose.

Reply:
I have dropped all the `in_` / `new_` prefixes from the interfaces, there are not needed.